scripts/test.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# This script is executed by Travis CI for every successful push (on any branch, PR or not).
+# It runs some basic tests on pages. If the build is also a PR, additional
+# checks are run through the check-pr script, and any message or error is sent
+# to tldr-bot to be commented on the PR.
+#
+# NOTE: must be run from the repository root directory to correctly work!
+# NOTE: `set -e` is applied conditionally only if needed.
+
+# Default test.
+function default_test {
+  markdownlint pages*/**/*.md
+  tldr-lint ./pages
+}
+
+# Special test for Travis CI pull request builds.
+# Runs default_test collecting errors for tldr-bot.
+function pr_test {
+  errs=`default_test 2>&1`
```

Comment:
Use `errs="$(default_test 2>&1)";` here. The backtick syntax is not obvious.

Reply:
Does it really? I've used it all around in the rest of the scripts. Are you suggesting to use `$(cmd)` everywhere instead of <code>`cmd`</code>?

Reply:
Some interesting discussion here: https://stackoverflow.com/questions/9449778/what-is-the-benefit-of-using-instead-of-backticks-in-shell-scripts

Reply:
Basically `$()` is actually POSIX

Reply:
I see, I shall refactor all the code to use `$()` then. Thanks guys.